scripts/e2e/Ingredients.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { test, expect } from '@playwright/test';
+
+test.describe('Recipe Page Tests', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('/');
+    await page.fill('input[name="username"]', 'bob_lee');
+    await page.fill('input[name="password"]', 'password');
+
+    // Click login button
+    await page.locator('button[type="submit"]').click();
+
+    // Wait for login response
+    await page.waitForResponse((response) => response.url().includes('/authenticate/login'));
+  });
+
+  test('Recipe creation', async ({ page }) => {
+    // Click the first Inventory button
+    await page.locator('a:has-text("Inventory")').nth(0).click();
+
+    // Click New ingredient button
+    await page.locator('button:has-text("New ingredient")').click();
+
+    // expect url to be inventory/new
+    expect(page.url()).toContain('/inventory/new');
+
+    await page.locator('input[placeholder="e.g. banana"]').fill('pepper');
+    await page.locator('input[placeholder="e.g. 35"]').fill('3');
+    await page.locator('#root__layout > div.layout__has__sidebar.MuiBox-root.css-7d7xpv > main > div > form > div.MuiGrid-root.MuiGrid-container.MuiGrid-spacing-xs-3.css-zow5z4-MuiGrid-root > div:nth-child(3) > div > div > input').fill('g');
+    await page.locator('input[type="date"].MuiInputBase-input.MuiOutlinedInput-input')
+        .fill('2024-11-18');
+    const addIngredientButton = page.locator('button:has-text("Add Ingredient")');
+    await page.locator('button:has-text("Add Ingredient")').waitFor({ state: 'visible' });
+    await addIngredientButton.click();
+    // wait for confirm button
+    const confirmButton = page.locator('button:has-text("Confirm")');
+    await confirmButton.waitFor({ state: 'visible' });
+    await confirmButton.click();
+  });
```

Comment:
clicking the confirm button isn't sufficient to test the successful creation of the ingredient. you need to make sure that there is a success message that pops up, and you need to navigate back to the ingredient table page and validate that the newly created ingredient is indeed in the table